contracts/test/GrantRoundManager.test.ts

```diff
@@ -210,25 +206,11 @@ describe('GrantRoundManager', () => {
       expect(await balanceOf('gtc', payee)).to.equal(amountOut);
     });
 
-    it.skip('input token DAI, output token GTC', async () => {
-      // TODO skipped because there is no DAI-GTC pool, so this fails as Uniswap V3 tries to swap directly in
-      // the specified pool. Need to determine how to specify a route
-
-      // Give the user 100 DAI
-      const amountIn = parseUnits('100', 18);
-      await setBalance('dai', user.address, amountIn);
-
-      // Set payee address to be a random address
-      const payee = randomAddress();
-      await mockRegistry.mock.getGrantPayee.returns(payee);
-
+    it('input token DAI, output token GTC', async () => {
```

Comment:
```suggestion
    it('input token DAI, output token GTC with no direct LP', async () => {
```

Reply:
Clarified in ed764701e4f064c93f2b6036e292895a3cf2f315